src/test/java/edu/hm/hafner/analysis/parser/CodeCheckerParserWithLinuxPathsTest.java

```diff
@@ -0,0 +1,55 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+import static edu.hm.hafner.analysis.assertions.Assertions.*;
+
+class CodeCheckerParserWithLinuxPathsTest extends AbstractParserTest {
+    private static final String WARNING_TYPE = "Warning";
+
+    CodeCheckerParserWithLinuxPathsTest() {
+        super("CodeChecker_with_linux_paths.txt");
+    }
+
+    @Override
+    protected CodeCheckerParser createParser() {
+        return new CodeCheckerParser();
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report annotation, final SoftAssertions softly) {
```

Comment:
Either use `report` or `annotations` (plural).
```suggestion
    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
```